nix/haskell.nix

```diff
@@ -147,10 +147,6 @@ haskell-nix: haskell-nix.cabalProject' [
         ] ++ lib.filter
           (drv: lib.isDerivation drv && drv.name != "regenerate-materialized-nix")
           (lib.attrValues haskell-build-tools));
-
-        CARDANO_NODE_CONFIGS = pkgs.cardano-node-deployments;
-
-        meta.platforms = lib.platforms.unix;
```

Comment:
There seem to be eval timeouts. This line might be related. https://input-output-rnd.slack.com/archives/C03NYMJBD7F/p1665772883659379?thread_ts=1665750580.794529&cid=C03NYMJBD7F